sdk/servicebus/azure-messaging-servicebus/swagger/src/main/java/AdministrationClientCustomization.java

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import com.azure.autorest.customization.ClassCustomization;
+import com.azure.autorest.customization.Customization;
+import com.azure.autorest.customization.LibraryCustomization;
+import com.azure.autorest.customization.PackageCustomization;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.FieldDeclaration;
+import com.github.javaparser.ast.expr.AnnotationExpr;
+import com.github.javaparser.ast.expr.MemberValuePair;
+import com.github.javaparser.ast.expr.NormalAnnotationExpr;
+import com.github.javaparser.ast.expr.StringLiteralExpr;
+import org.slf4j.Logger;
+
+import java.util.HashSet;
+import java.util.Objects;
+
+/**
+ * Customization class for Administration client for Service Bus
+ */
+public class AdministrationClientCustomization extends Customization {
```

Comment:
thanks for the Customization type, it saves so much time (also nice to see detailed comments on this class.)